### PR TITLE
Laboratory work #8, Sviridov Vyacheslav - 20FPL1

### PR DESCRIPTION
**Pull request by VSvir**

### CODE REVIEW

File: lab_7_llm/assets/index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```
Comment (SalnikovIgor) [inline lab_7_llm/assets/index.html]: Remove all files for lab 7 from the PR.

**Review by SalnikovIgor — CHANGES_REQUESTED** [review]

File: lab_7_llm/README.rst [hunk 2]
```diff
@@ -1,729 +0,0 @@
-.. _lab_7_llm/README:
```
Comment (SalnikovIgor) [inline lab_7_llm/README.rst]: Too radical an approach to PR cleanup. The lab 7 files must be rolled back to the current main branch

**Review by SalnikovIgor — CHANGES_REQUESTED** [review]

File: lab_7_llm/README.rst (hunk 2 above)
Comment (VSvir) [reply lab_7_llm/README.rst]: Well, if _remove all files_ doesn't mean to remove all files...

Comment (SalnikovIgor) [reply lab_7_llm/README.rst]: I agree, need to be more careful in the motives for action

File: lab_7_llm/main.py [hunk 3]
```diff
@@ -4,7 +4,7 @@
 # pylint: disable=too-few-public-methods, undefined-variable, too-many-arguments, super-init-not-called
```
Comment (SalnikovIgor) [inline lab_7_llm/main.py]: But why are these changes from lab  7 still here?

**Review by SalnikovIgor — CHANGES_REQUESTED** [review]
End checkout lab 7 to master and add solutions for mark 8 and 10

File: lab_7_llm/main.py (hunk 3 above)
Comment (VSvir) [reply lab_7_llm/main.py]: It should be ok now

**Comment by VSvir** [comment]
Yay🎉

**Pull request closed without merging**